src/clearwater_modules/nsm1/n2/processes.py

```diff
@@ -82,16 +82,18 @@ def dN2dt(
 def N2(
     N2: xr.DataArray,
     dN2dt : xr.DataArray,
+    timestep: xr.DataArray
 ) -> xr.DataArray: 
     
     """Calculate change in N2 air concentration (mg-N/L/d)
 
     Args:
         N2: Nitrogen concentration air (mg-N/L)
         dN2dt: Change in nitrogen concentration air
+        timestep: Current iteration timestep (d)
```

Comment:
@sjordan29, you make a very important point. Time handling can be quite complicated, because of leap seconds, etc. It would be much better to have a Python library handle it, because those contain all the necessary considerations and conversions.

Let's create a new issue on this topic, to address in a future effort.

Reply:
In the meanwhile, we need a time step solution for now. @sjordan29, when you ran TSM, I know you ran the model at a 1-second time step, but how did you deal with rate coefficients that used a different time unit?

Reply:
@sjordan29 the timestep variable is supposed to be dt as you describe it in the original comment - the size of the timestep. Can change the variable name to make that more clear.

@aufdenkampe for dealing with rate coefficients that use a different unit time, it was basically assumed that the fortran code made all the necessary conversions, such that if our model matched the TSM unit tests we considered it to be accurately calculating that change in temp. We should probably do a unit analysis.

Reply:
Got it -- I see that in the global variables file. I think we should change it to `dt` or `delta_time` to avoid confusion now that we are explicitly tracking the timestep as part of #77.

Reply:
Ok sounds good. I can do that tomorrow morning. Or if you think more thought needs to be put into the `dt` variable I will wait. I was envisioning this being a state variable eventually, since it will be information supplied by the main engine.

Reply:
Opened issue #79